examples/src/content/Swappable/Floated/index.js

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable import/no-extraneous-dependencies, import/no-unresolved */
-import {Swappable} from '@shopify/draggable';
+import {Swappable, Plugins} from '@shopify/draggable';
```

Comment:
Added to this example as it works better than the MultipleContainers example...

You may notice that the `height` in MultipleContainers changes instantly (does not transition). This is some CSS bullshit no doubt. So... I might make all of these Swappable examples use the ResizeMirror plugin.